.chloggen/metricsgeneration_metric2_0.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
I consider this to be a breaking change as existing collector configurations will now potentially be emitting more metrics than before.